build/mac/BUILD.gn

```diff
@@ -2,11 +2,11 @@ import("//brave/build/config.gni")
 
 declare_args() {
   mac_signing_identifier = ""
-  mac_signing_keychain = "login"
+  mac_signing_keychain = "keychain"
```

Comment:
This should be the name of the keychain, login is usually the default so that makes sense here still I think.